echopop/core.py

```diff
@@ -7,6 +7,71 @@
 import numpy as np
 import pandas as pd
 
+# Required configuration initialization YAML structure
+CONFIG_INIT_API = {
```

Comment:
I feel using API here is a bit strange. How about we borrow the pydantic lingo to use "model"?
```suggestion
CONFIG_INIT_MODEL = {
```